test/tests-workspaces.cpp

```diff
@@ -516,8 +516,8 @@ TEST(workspaces, DISABLED_tile_dotProduct_2) {
   Tensor<double> C("C", {N}, Format({Dense}));
 
   for (int i = 0; i < N; i++) {
-    B.insert({i}, (double) i);
-    C.insert({i}, (double) i);
+    B.insert({i}, (double) i / N);
```

Comment:
Why was this necessary?

Reply:
Because current TACO checking function can't correctly check the equality between two relatively large numbers. If B's content is not divided by N, the test will fail even if it is correct